static/app/views/performance/table.tsx

```diff
@@ -451,7 +471,15 @@ class _Table extends Component<Props, State> {
         </GuideAnchor>
       );
     }
-    return sortLink;
+
+    return tooltip ? (
+      <Header>
+        {sortLink}
+        <StyledQuestionTooltip size="xs" position="top" title={tooltip} isHoverable />
+      </Header>
+    ) : (
+      sortLink
+    );
```

Comment:
The more I code, the more i feel like complex ternary's should just be simple if statements. I find it's easier to read
```suggestion
    if (!tooltip) {
       return sortLink
    }
    return (<Header>
        {sortLink}
        <StyledQuestionTooltip size="xs" position="top" title={tooltip} isHoverable />
      </Header>)
```